src/io/RestoreH5MDParallel.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2015-2018
```

Comment:
Please update the header, put your first, last name and institute (if applicable)

Reply:
fixed